.github/workflows/tests.yml

```diff
@@ -17,28 +17,20 @@ jobs:
         - ["2.7",   "py27-msgpack1"]
         - ["2.7",   "py27-zeo4"]
         - ["2.7",   "py27-zodbmaster"]
-        - ["3.5",   "py35"]
```

Comment:
All other zopefoundation packages which still support Python 2.7 also support 3.5 as Zope 4 supports both of them. Is there another reason for removing Python 3.5 support besides that is old and dead?

Reply:
I am not "removing Python 3.5 support", I am just removing the test for it on GitHub Actions. Those tests burn thousands of useless minutes for a Python version that is EOL.

Reply:
Okay, let's stop testing Python 3.5. Even though I cannot get rid of the feeling that not testing it means not longer supporting it: We will not see problems arising with that version as time goes by.

Reply:
Look at it this way: If anyone comes to us with a problem report for Python 3.5 we will definitely help. That's sufficient to claim support.

Reply:
Okay, convinced.